utils/sql-util/src/main/java/io/flamingock/internal/common/sql/SqlVendor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2025 Flamingock (https://www.flamingock.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.flamingock.internal.common.sql;
+
+public enum SqlVendor {
```

Comment:
I am wondering if we should name this enum as SqlDialect and the public method to parse the vendor as "fromDatabaseProductName" (because is the name of the property we are using from Datasource).